webapp/application.py

```diff
@@ -18,6 +22,8 @@
 session = talisker.requests.get_session()
 harvest = Harvest(session=session, api_key=os.environ.get("HARVEST_API_KEY"))
 cipher = Cipher(os.environ.get("APPLICATION_CRYPTO_SECRET_KEY"))
+verification_token_cipher = Cipher(os.environ.get("SECRET_KEY"))
```

Comment:
I'd like two understand the distinct things these two different ciphers are doing. I'll try to find time to delve into the code more to look into it, unless you're able to give me a quick summary.